manage.py

```diff
@@ -24,20 +25,52 @@
 app = create_app()
 manager = Manager(app)
 
-migrate = Migrate(
-    app,
-    db,
-    directory=os.path.join(app.root_path, 'migrations')
-)
+MIGRATIONS_DIR = os.path.join(app.root_path, 'migrations')
+migrate = Migrate(app, db, directory=MIGRATIONS_DIR)
 manager.add_command('db', MigrateCommand)
 manager.add_command('runserver', ConfigServer(host='0.0.0.0', threaded=True))
 
 
+def _run_alembic_command(args):
+    """Helper to manage working directory and run given alembic commands"""
+    # Alembic looks for the alembic.ini file in CWD
+    # hop over there and then return to CWD
+    cwd = os.getcwd()
+    os.chdir(MIGRATIONS_DIR)
+
+    alembic.config.main(argv=args)
```

Comment:
I think you can pass the path to alembic.ini as an arg (or kwarg `file_`) to the[ `Config` class](https://github.com/briandailey/alembic/blob/master/alembic/config.py#L23) instead of `cd`ing... but I don't think there's a functional difference